src/main/webapp/app/exercises/shared/result/result-detail.component.html

```diff
@@ -1,25 +1,31 @@
-<div class="modal-header">
-    <h4 class="modal-title">
+<div class="modal-header feedback-header">
+    <h3 class="modal-title">
         <ng-container *ngIf="taskName; else noTaskDescription">{{ 'artemisApp.result.detail.feedbackForTask' | artemisTranslate : { taskName } }}</ng-container>
         <ng-template #noTaskDescription>{{ 'artemisApp.result.detail.feedback' | artemisTranslate }}</ng-template>
-    </h4>
-    <button type="button" class="btn-close" data-dismiss="modal" aria-hidden="true" (click)="activeModal.close()"></button>
+    </h3>
+
+    <button type="button" class="feedback-header-close" data-dismiss="modal" aria-hidden="true" (click)="activeModal.close()">
+        <fa-icon [icon]="faXmark" size="xl"></fa-icon>
```

Comment:
We cant use bootstrap `btn-close` here because we wouldn't have any control over the styles then

Reply:
@JohannesStoehr please check that I didn't break anything during a merge conflict with [5851](https://github.com/ls1intum/Artemis/pull/5851)